From c24fbe335c22ced933edb7d72f03adaf64000056 Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Tue, 20 Aug 2013 15:09:47 +0200
Subject: [PATCH 19/28] pc: Remove incorrect rhel6.x compat "model" value for Conroe/Penryn/Nehalem

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1377011392-9336-3-git-send-email-ehabkost@redhat.com>
Patchwork-id: 53610
O-Subject: [RHEL7 PATCH 2/7] pc: Remove incorrect rhel6.x compat "model" value for Conroe/Penryn/Nehalem
Bugzilla: 918907
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

Bugzilla: 918907
Upstream status: not applicable

The series that introduced the rhel6.x machine-types added compat
"model" values for Conroe, Penryn and Nehalem that were copied from
PC_COMPAT_1_5 but are not applicable to rhel6.x, as the CPU models had
the correct model value since RHEL-6.0 (see RHEL-6 commit
9a2e04fed8d1c4b44c4122b211e236a36a036cc7).

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/i386/pc_piix.c | 12 ------------
 1 file changed, 12 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |   12 ------------
 1 files changed, 0 insertions(+), 12 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 0dc52c4..a11a4b0 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -910,26 +910,14 @@ static QEMUMachine pc_machine_rhel640 = {
     PC_RHEL6_4_COMPAT,\
     {\
         .driver   = "Conroe-" TYPE_X86_CPU,\
-        .property = "model",\
-        .value    = stringify(2),\
-    },{\
-        .driver   = "Conroe-" TYPE_X86_CPU,\
         .property = "level",\
         .value    = stringify(2),\
     },{\
         .driver   = "Penryn-" TYPE_X86_CPU,\
-        .property = "model",\
-        .value    = stringify(2),\
-    },{\
-        .driver   = "Penryn-" TYPE_X86_CPU,\
         .property = "level",\
         .value    = stringify(2),\
     },{\
         .driver   = "Nehalem-" TYPE_X86_CPU,\
-        .property = "model",\
-        .value    = stringify(2),\
-    },{\
-        .driver   = "Nehalem-" TYPE_X86_CPU,\
         .property = "level",\
         .value    = stringify(2),\
     },{\
-- 
1.7.1

